boa/src/syntax/parser/expression/mod.rs

```diff
@@ -61,7 +61,7 @@ macro_rules! expression { ($name:ident, $lower:ident, [$( $op:path ),*], [$( $lo
     {
         type Output = Node;
 
-        fn parse(self, cursor: &mut Cursor<R>) -> ParseResult {
+        fn parse(self, cursor: &mut Cursor<R>)-> ParseResult {
```

Comment:
Surprised that `rustfmt` removed the space before the arrow. But it's passing CI so it must have...

Reply:
Huh, that is a bit strange